pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "pytom-template-matching-gpu"
-version = "0.3.4"
+version = "0.3.5"
```

we never did a 0.3.4 release, so this doesn't have to change
```suggestion
version = "0.3.4"
```

---

ah okay